source/index.html.md

```diff
@@ -25,6 +25,11 @@ system functions, along with examples in common programming languages.
 
 # Changelog
 
+### 2021-07-19
+Documentation inconsistency between REST Api service and implementation was fixed:
+
+* in user_trades service documentation, now 'book' is not required.
```

Comment:
Does this correctly render as list? I've seen some list in API doc that doesn't look good.

Reply:
The 2021-06-24 one has poor formatting.

Reply:
yes, it is fixed: https://stage.bitso.com/api_info